heatintro.md

```diff
@@ -1,85 +1,142 @@
+          metadata: {"scale_group": {get_param: "OS::stack_id"}}
+          networks:
+          - network: private
+          security_groups:
+          - testvm
 
-Some examples of resource:
 
-Some of the resouce groups relevant today for us is:
+---
+## Some of the resouce groups relevant today for us is:
 - *OS::Heat::AutoScalingGroup*
 - *OS::Heat::ScalingPolicy*
 - *OS::Monasca::Notification*
 - *OS::Monasca::AlarmDefinition*
 
 
-- *OS::Heat::AutoScalingGroup*
-  Properties:
-  - name: Convenient name.
-  - max_size: Maximum size of the group.
-  - min_size: Minimum size of the group.
-  - cooldown: minimum time before *any* autoscaling related operation.
-  - resources: set of resource that should be scaled up or down.
-
-- Heat provdes this group a box of resources that can be scaled.
-- Scaling up and down is based on Scaling Policy for the group.
-- Should be under a group, because the policy has to back refer to it
-
-
-*OS::Heat::ScalingPolicy*
-    Properties:
-    - change: a number that has an effect based on change_type.
-    - change_type: (meaning of "change")
-        - "change_in_capacity"
-        - "percentage_change_in_capacity", 
-        - "exact_capacity" 
-    - cooldown: minimum amount of time (in seconds) between allowable executions of this policy.
-    - group_id: ID of the group that this policy will affect
-    - name: Convenient name
-
-    Attributes: (outputs)
-    - alarm_url: the url which triggers the policy
-
-
-*OS::Monasca::Notification*
-    Properties:
-    - address: Address of the notification. 
+---
+## OS::Heat::AutoScalingGroup
+Properties:
+  - **name:** Convenient name.
+  - **max_size:** Maximum size of the group.
+  - **min_size:** Minimum size of the group.
+  - **cooldown:** minimum time before *any* autoscaling related operation.
+  - **resources:** set of resource that should be scaled up or down.
+
+Example:
+
+    group:
+      type: OS::Heat::AutoScalingGroup
+      properties:
+        ... specify propoerties from above ..
+        resource:
+          type: OS::Nova::Server
+          properties:
+            flavor: { get_param: flavor }
+            image: { get_param: image }
+            # pass scale_group as metadata,
+            # monasca will store this as vm metadata and
+            # later can be used to group vms together from
+            # same stack
+            metadata: {"scale_group": {get_param: "OS::stack_id"}}
+            networks:
+            - network: private
+            security_groups:
+            - testvm
+
+---
+## OS::Heat::ScalingPolicy
+Properties:
+  - **change**: a number that has an effect based on change_type.
+  - **change_type:** (meaning of "change")
+  	- "change_in_capacity"
+    - "percentage_change_in_capacity", 
+    - "exact_capacity" 
+  - **cooldown:** minimum amount of time (in seconds) between allowable executions of this policy.
+  - **group_id:** ID of the group that this policy will affect
+  - **name:** Convenient name
```

Comment:
What about "auto_scaling_group_id" and "scaling_adjustment" properties which we list in the example?